m4_sessions/1903_inning/automation.md

```diff
@@ -1,53 +1,48 @@
 ## Group Eight's Session Plan
 
-M4 Student Leaders: Mike, Justin, Jesse, Ethan, Dan
+M4 Student Leaders: Justin, Leah, Matthew, Nick 
 
-# Turing Career Development: Automation
+# Turing Career Development: Automation: Job Elimination 101
 
 ## Session Outcomes
 
-## Mission Alignment		 
-  		  
-*Our mission is to unlock human potential by training a diverse, inclusive student body to succeed in high-fulfillment technical careers.
-  		  
-As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
+As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission at Turing is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
 
-## Pre-Session
+#### Introductions 1:00 - 1:05
+  * Have students introduce themselves
+    Name:
+    CoHort: 
+    Pronouns:
+    Answer Questions:
+   1. If you could have one process in your life automated tomorrow what would it be?
+   2. What automated process today do you appreciate the most?
 
+#### Icebreaker 1:05 - 1:10
+   Automation icebreaker?
 
-### Survey Questions
-Survey to be sent either prior to session or immediately after ( prior may lead to better response? )
+### Groupings 1:10 - 1:15
+  We will group members by opposing stances. Have students stand in a spectrum of how much they like the idea of automation, and from that hand select one each from opposing sides and a middle. This should facilitate the most conversation and opposing viewpoints.
 
-#### Survey to lighten the mood
-  * funny questions like:
-   1. When the robots attack, what’s the first thing you do?
-   2. If you could have one process in your life automated tomorrow what would it be?
-   3. What automated process today do you appreciate the most
-   4. Write your apology to the person whose job you stole by automating that thing.
-   
-### Pre-Reading
+#### Whiteboard Pros and Cons 1:15 - 1:25
+  Have groups write pros and cons on board or papers.
+  Each group shares a pro and con  
+  
+### POM 1:25
 
-Send pre-reading out at least 24 hours before the session
+#### Watch Video 1:30 - 1:45
```

Comment:
Which video are they watching here?  The full 15 minutes?  

I would share the questions ahead of the video so they can start thinking about the questions.  Also please turn on closed captioning for the video!